example/ios/Demo/AppDelegate.swift

```diff
@@ -43,7 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let vc = storyboard.instantiateViewController(withIdentifier: "LinksViewController") as! LinksViewController
         self.window?.rootViewController = vc
         
-        vc.handleIncomingLink(url: stringUrl)
+//        vc.handleIncomingLink(url: stringUrl)
```

Comment:
I suspect this is broken now.

Reply:
Yeah ignore for now, I think I temp disabled it to get things to compile on Swift. Its properly fixed in the SwiftUI example branch